nilearn/plotting/surf_plotting.py

```diff
@@ -1623,10 +1623,15 @@ def plot_surf_roi(surf_mesh, roi_map, bg_map=None,
 
     if cbar_tick_format == "auto":
         cbar_tick_format = "." if engine == "plotly" else "%i"
+
+    symmetric_cmap = kwargs.get("symmetric_cmap", False)
+    if kwargs.get("symmetric_cmap"):
```

Comment:
```suggestion
    if kwargs.get("symmetric_cmap") is not None:
```

so the test passes if set to False